### PR TITLE
[IMP]Event: add config for event barcode

### PR DESCRIPTION
**Pull request by mba-odoo**
Task: https://www.odoo.com/web#id=20855&view_type=form&model=project.task&action=327
Pad: https://pad.odoo.com/p/r.Q5AfjrQcEpE6NPRT

### CODE REVIEW

File: addons/event/models/event_config_settings.py [hunk 1]
```diff
@@ -31,7 +31,9 @@ class event_config_settings(osv.TransientModel):
             (1, 'Schedule emails to attendees and subscribers')
             ], "Email Scheduling",
             help='You will be able to configure emails, and to schedule them to be automatically sent to the attendees on subscription and/or attendance',
-            implied_group='event.group_email_scheduling'),            
+            implied_group='event.group_email_scheduling'),
+        'module_event_barcode': fields.boolean("Scan badges to confirm attendances",
```
Comment (mart-e) [inline addons/event/models/event_config_settings.py]: put this as a selection field

Comment (mba-odoo) [inline addons/event/models/event_config_settings.py]: @mart-e  , i have already put the selection but we can not apply widget upgrade_boolean(means two widget radio and upgrade_boolean) on Selection and @bst-odoo  said inn pad put enterprise upgrade button on that so i make it boolean :  https://drive.google.com/a/openerp.com/file/d/0BxqHe5rtIxY0cExNeXNJd19adjA/view?usp=drivesdk  thanks

Comment (mart-e) [inline addons/event/models/event_config_settings.py]: @mba-odoo ok, thanks for the info

File: addons/event/report/event_event_templates.xml [hunk 2]
```diff
@@ -26,6 +26,8 @@
                                             <h3 t-field="o.name"/>
                                         </small>
                                     </div>
+                                     <div class="col-xs-12 text-center" id="o_event_barcode">
```
Comment (tde-banana-odoo) [inline addons/event/report/event_event_templates.xml]: Nope. Everything should stay modular.

File: addons/event/report/event_event_templates.xml [hunk 3]
```diff
@@ -76,6 +78,8 @@
                                 <div class="col-xs-12 text-center" id="o_event_attendee_name">
                                     <small><h3>Attendee Name</h3></small>
                                 </div>
+                                <div class="col-xs-12 text-center" id="o_event_barcode">
```
Comment (tde-banana-odoo) [inline addons/event/report/event_event_templates.xml]: Nope. Everything should stay modular.

File: openerp/api.py [hunk 4]
```diff
@@ -348,6 +348,36 @@ def old_api(self, cr, uid, *args, **kwargs):
     return make_wrapper(model, method, old_api, method)
 
 
+def almostmodel(method):
```
Comment (tde-banana-odoo) [inline openerp/api.py]: We should probably keep this name. Or not. Not sure.

Comment (mart-e) [inline openerp/api.py]: `@api.probablynotamodelwhocares`

File: addons/event_sale/report/event_event_templates.xml [hunk 5]
```diff
@@ -2,17 +2,21 @@
 <odoo>
 
     <template id="event_registration_report_template_badge" inherit_id="event.event_registration_report_template_badge">
-        <xpath expr="//div[@id='o_event_name']" position="after">
-            <div t-if="o.event_ticket_id" class="col-xs-12 mt32 text-center" style="background: lightgrey; height: 65px">
-                <h3><span t-field="o.event_ticket_id"/></h3>
+        <xpath expr="//div[@id='o_event_name']" position="inside">
```
Comment (tivisse) [inline addons/event_sale/report/event_event_templates.xml]: Why did you modify the badge template ? To leave more space for the barcode ?

**Pull request merged**